src/llama.cpp

```diff
@@ -2538,6 +2565,14 @@ struct llama_hparams {
             return ssm_d_state * ssm_d_inner;
         }
     }
+
+    bool n_bskcn(uint32_t n, uint32_t il = 0) const {
```

Comment:
The `n_` prefix implies that this returns an integer, however it returns a boolean.